pages/common/tldr-lint.md

```diff
@@ -0,0 +1,16 @@
+# tldrl
```

Comment:
`s/tldrl/tldr-lint` through this file?

Reply:
Sorry about that, just another copy/paste error 🙂